### PR TITLE
Adding Orange as dependency

### PR DESCRIPTION
**Pull request by nikicc**
Since Orange3 came on pip it can be added among our requirements.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
 INSTALL_REQUIRES = sorted(set(
     line.partition('#')[0].strip()
     for line in open(os.path.join(os.path.dirname(__file__), 'requirements.txt'))
-) - {''})
+) - {'', 'https://github.com/biolab/orange3/zipball/master'})
```
Comment (kernc) [inline setup.py]: I don't think I like this. :smiley:

Comment (nikicc) [inline setup.py]: Me neither. Any suggestions?

**Comment by ajdapretnar** [comment]
@nikicc @kernc I'm again nazing around on issue trackers, so... We discussed on Friday to simply add Orange to pip once and for all. Would this solve the problem? If so, we probably want to push the pip fix high on our priority list (as I think we should).

**Comment by nikicc** [comment]
Not sure if it would solve this, but it would make it much easier to solve for sure. 

I suggest we wait till it is on pip and then I will refactor this PR so that documentation will work.

File: .travis.yml [hunk 2]
```diff
@@ -29,7 +29,6 @@ before_install:
     - pip install codecov
 
 install:
-    - travis_wait source $TRAVIS_BUILD_DIR/travis/install_orange.sh
```
Comment (kernc) [inline .travis.yml]: This file/dir still needed?

Comment (nikicc) [inline .travis.yml]: Nop, I was just sloppy. Should be gone now.

**Pull request merged**